bulletroboy/forces_mapper.py

```diff
@@ -0,0 +1,156 @@
+import rclpy
+from rclpy.node import Node
+
+# from sensor_msgs.msg import JointState
+from roboy_simulation_msgs.msg import Collision
+import yaml
+from roboy_simulation_msgs.srv import LinkIdFromName
+from roboy_simulation_msgs.srv import LinkNameFromId
+from asgiref.sync import async_to_sync
+
+class ForcesMapper(Node):
+    def __init__(self):
+        super().__init__('forces_imitator')
+
+        self.result = None
+
+        self.robotCollisionSubscription = self.create_subscription(
+            Collision,
+            '/roboy/simulation/collision',
+            self.collision_listener,
+            10)
+
+        self.link_names_map = self.load_roboy_to_human_link_name_map()
+
+        # Define clients
+        self.roboy_link_name_from_id_client = self.create_client(LinkNameFromId, '/roboy/simulation/roboy/link_name_from_link_id')
+        self.operator_link_id_from_name_client = self.create_client(LinkIdFromName, '/roboy/simulation/operator/link_id_from_link_name')
+
+        # Connect to services
+        while not self.roboy_link_name_from_id_client.wait_for_service(timeout_sec=1.0):
+            self.get_logger().info('service not available, waiting again...')
+        
+        while not self.operator_link_id_from_name_client.wait_for_service(timeout_sec=1.0):
+            self.get_logger().info('service not available, waiting again...')
+
+        self.exoforceCollisionPublisher = self.create_publisher(Collision, 'roboy/exoforce/collisions', 10)
+        timer_period = 0.005 # seconds
+        self.time = self.create_timer(timer_period, self.timer_callback)
```

Comment:
Same as for self.robotCollisionSubscription. self.time as a variable name isnt used again.